seeds/communities.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+exports.seed = function(knex, Promise) {
+    return Promise.join(
```

Comment:
note however that all promises passed to Promise.join are run concurrently, so you may want to do `del().then(() => Promise.join(...))` to avoid the occasional race condition